src/python/pants/backend/python/packaging/pyoxidizer/config.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
I'm completely skeptical of this whole file, but I'm not sure what a better approach is. Should there even be a "default" configuration if the user doesn't pass one in? There are a ton of configuration options - and while this is the most reasonable by default (and thus, probably the easiest for a new user to start with) - it feels a bit dicey too.

Reply:
The template is quite a bit of boiler plate, so a basic example to get going is reasonable to have, I think.

Reply:
It's tricky to say.

One potential razor to help decide whether the plugin should attempt to lean in and provide a default template might be whether we think that we can maintain a template over time that is able to build an artifact from most distributions using a reasonable number of arguments on the target to control behavior.

For example: if we think that by exposing less than a dozen options (`unclassified_resources=`, etc) from `pyoxidizer_binary` we can build 90% of distributions, then it might be worth trying.

I really don't know. But one thing that would seem to be in favor of that approach is that installing from a distribution seems like a _relatively_ self-contained situation (unlike building from loose sources and dynamically adding requirements).